### ISSUE TITLE
Is there a way to create checkbox input from LDAP_ACCOUNT_ADDITIONAL_ATTRIBUTES?

### ISSUE THREAD

**Issue by dbrw** [wheelybird/ldap-user-manager#203]
I want to be able to add mailEnabled attribute in user creation and edit, but couldn't found how to do it. I assume it should have checkbox input type that translate to true/false in ldap.

Any suggestion on how to do it or handle it?

**Comment by wheelybird** [comment]
Hi.  Currently this isn't possible.  The only way I think you could do this at the moment is to add mailEnabled as an attribute with a default of either `TRUE` or `FALSE` and update accordingly when adding/editing accounts.   It's a good suggestion though, so I'll look into adding checkboxes in a future release.

**Comment by dbrw** [comment]
Yes, for now my solution is use TRUE as default value for the input field. Please note the uppercased value for anyone new to openldap.
